### DISCUSSION TITLE
Trying to make lootable containers, but I'm lost

### DISCUSSION THREAD

**Discussion by lazar-the-coder** [peter-kish/gloot#119] [Q&A]
The container is an object in the world. The player inventory is connected to the UI singleton. I'm not sure how to connect the inventory on the container to the ctrl on the ui
![Screenshot 2023-09-13 193934](https://github.com/peter-kish/gloot/assets/120413005/0874a08e-28c9-42d3-84b2-5c016903bf1c)
![Screenshot 2023-09-13 194031](https://github.com/peter-kish/gloot/assets/120413005/310168e2-5080-4a39-8bc9-684ea974f54b)

**Comment by peter-kish** [comment]
You should be able to do that from by setting the `inventory` property of the [`CtrlInventory`](https://github.com/peter-kish/gloot/blob/master/docs/ctrl_inventory.md) class. The same should work with [`CtrlInventoryStacked`](https://github.com/peter-kish/gloot/blob/master/docs/ctrl_inventory_stacked.md), as it inherits from `CtrlInventory`.

Something like:
```gdscript
ctrl_inventory_stacked.inventory = container.inventory_stacked
```

**Reply by lazar-the-coder** [reply]
Thank you soooo much
